src/test/java/org/devcon/ticket/UseTicketTest.java

```diff
@@ -68,26 +68,26 @@ public void testSunshine() {
     try {
       PublicKey pk;
       System.out.println("Signed attestation:");
-      System.out.println(DERUtility.printDER(attestedTicket.getAtt().getDerEncoding(), "SIGNABLE"));
+      System.out.println(DERUtility.toPEM(attestedTicket.getAtt().getDerEncoding(), "SIGNABLE"));
```

Comment:
The tests now only print out the object reference and not the actual PEM file in human readable base 64.